content/blog/2017-09-04-promcon-2017-recap.md

```diff
@@ -0,0 +1,122 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Speakers from [Grafana](https://grafana.com/) and [InfluxData](https://www.influxdata.com/) brought us up to date with new features and Prometheus integrations:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Next, <a href="https://twitter.com/CarlBergquist">@CarlBergquist</a> from <a href="https://twitter.com/raintanksaas">@raintanksaas</a> shows cool tricks and new features in <a href="https://twitter.com/grafana">@grafana</a>. <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/aXCbOAA1dp">pic.twitter.com/aXCbOAA1dp</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898119125540765697">August 17, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">You like Prometheus but have a need for long-term storage? <a href="https://twitter.com/pauldix">@pauldix</a> presents how <a href="https://twitter.com/InfluxDB">@InfluxDB</a> integrates with Prometheus at <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/hpsO95mPVL">pic.twitter.com/hpsO95mPVL</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898476044327497728">August 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Several Prometheus core developers also spoke about best practices and new features and developments in Prometheus:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Our own <a href="https://twitter.com/juliusvolz">@juliusvolz</a> gives a lecture on best practices and pitfalls when starting with <a href="https://twitter.com/PrometheusIO">@PrometheusIO</a> <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/gTNrxeaxK7">pic.twitter.com/gTNrxeaxK7</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898098812761362432">August 17, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">. <a href="https://twitter.com/mxinden">@mxinden</a> from <a href="https://twitter.com/coreos">@coreos</a> talks about how and why the alertmanager got it&#39;s shiny new UI. <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/zXaZS9rXqc">pic.twitter.com/zXaZS9rXqc</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898525224924327936">August 18, 2017</a></blockquote>
```

Comment:
its, as it is possessive.

Reply:
Yes, but it's a tweet which we can't change - changing the copy of it here doesn't seem good.